cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -265,23 +269,29 @@ get_cross_toolchain_pkg() {
 
 # Download, extracts and install the toolchain package
 install_cross_toolchain_pkg() {
-  mkdir -p /build
-  pushd /build
-
-  info "Downloading toolchain from ${TOOLCHAIN_DOWNLOAD_URL}"
   local -r pkg_name="$(basename "${TOOLCHAIN_DOWNLOAD_URL}")"
```

Comment:
You can put this line before line 283 becuse pkg_name is not used here.

Reply:
Thanks for your comments! Updated.